mlir/lib/Dialect/SparseTensor/Transforms/Sparsification.cpp

```diff
@@ -498,9 +498,17 @@ static Value genTensorLoad(CodegenEnv &env, OpBuilder &builder, ExprId exp) {
   Value val = env.exp(exp).val;
   if (val)
     return val;
-  // Load during insertion.
+  // Get tensor operand.
   linalg::GenericOp op = env.op();
+  Location loc = op.getLoc();
   OpOperand *t = &op->getOpOperand(env.exp(exp).tensor);
+  // Fold binary-valued tensor into explicit value.
+  const auto stt = getSparseTensorType(t->get());
+  if (stt.hasEncoding()) {
+    if (auto explVal = stt.getExplicitVal())
```

Comment:
nit: Can we implement `stt::getExplicitVal` as `return enc ? enc.getExplicitVal() : nullptr` to fuse two if statements?

Reply:
We think alike, I noticed that too. Seems cleaner indeed.